app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,20 @@
+class MoviesController < ApplicationController
+
+  def show
+    @movie = Movie.find(params[:id])
+    @actor_names = @movie.order_by_age
+    @avg_age = @movie.avg_age
+    if params[:search_by_name].present? 
```

Comment:
You made this US a bit more complicated (and more similar to the US from the pair project) -- you actually don't need to search for an actor and show the possible actors someone could add. Instead, you just needed to create a form that took an actor ID and routed to a controller action that would associate that actor with the given movie. No need for search -- the user just had to know the actor ID it was interested in adding